fix: docs fixes for client installation (#8461)
DIRACGrid/DIRAC: docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

  $ pip install DIRAC

will install the most recent production versions found in https://pypi.org/project/DIRAC/. This will also install the most recent versions of DiracX.

> You should now be able to tab-complete `dirac-`` to see all of the DIRAC commands that are available.

Reviewer: ```suggestion
You should now be able to tab-complete `dirac-` to see all of the DIRAC commands that are available.
```

At this point, you have a DIRAC/DiracX client installed, but the client needs to be connected to a server.
The configuration to connect to the server will be written down (in a "dirac.cfg" file) by the `dirac-configure` command.
Before running it, make sure that the following conditions are satisfied:

> * As a user, you have a personal certificate in `.pem` format. Such certificate is stored in the `~/.globus/` directory of the machine.

Reviewer: ```suggestion
* As a user, you have a personal certificate in `.pem` format. This certificate must be stored in the `~/.globus/` directory of the machine.
```

* You are part of a Virtual Organization. You are registered in your VO's VOMS server (for talking to DIRAC), and in your VO's IdP server (e.g. IAM) (for talking to DiracX).
* Your VO admins have set up a DIRAC/DiracX installation to which to connect to.

For the configuration, simply issuing `dirac-configure` will start a configuration wizard, using some predefined setups. If your setup is not present, but you know the URL of DIRAC's Configuration Server, issue instead::

  $ dirac-configure -C $DIRAC_CS_URL --SkipCAChecks --SkipCADownload

In both cases you will be prompted to insert the password of your personal certificate.

Using a user proxy
==================

At this point you need a proxy, so you should issue the command::

   $ dirac-proxy-init

> You can see which file is your proxy certificate using the `dirac-proxy-info` command. This command will also embed a token in the proxy, in order to talk to DiracX.

Reviewer: "This command" here would refer to "dirac-proxy-info", which probably does not embed a token in the proxy?
